README.rst

```diff
@@ -325,6 +325,23 @@ any feature requests will be prioritized according to
 what maintainers encounter as a need in our day-to-day jobs.
 Please temper expectations accordingly.
 
+API Policy
+~~~~~~~~~~
+
+``pyjanitor`` only extends or aliases the pandas API
+(and other dataframe APIs),
+but will never fix or replace them.
+
+Undesirable ``pandas`` behaviour should be reported upstream
+in the ``pandas`` repository.
```

Comment:
Maybe we can add a hyperlink to `pandas` [issues page](https://github.com/pandas-dev/pandas/issues)?

Reply:
Also a very good idea. Thanks @hectormz!